vendor/k8s.io/kubernetes/pkg/controller/podautoscaler/horizontal.go

```diff
@@ -360,6 +360,20 @@ func (a *HorizontalController) reconcileAutoscaler(hpav1Shared *autoscalingv1.Ho
 	}
 
 	mappings, err := a.mapper.RESTMappings(targetGK)
+	// TODO(directxman12): this is a dirty, dirty hack because legacy oapi isn't part of
```

Comment:
Here's a weird niggle. To make rebases easier, please move the logic to a file called `patch_dc.go` and then after the error simply do `mappings = hackOutOapiDeploymentConfig(mappings)`

That way when it conflicts (and there are always conflicts), we have a single line to fix up

Reply:
ah, ack, will do.